### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by kennethvdberghe**

### CODE REVIEW

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks for this. I was intentional about the HTML extension. I'm talking about adding a loading state that shows up instantly without the need for JavaScript to load. How could we change this to make it more clear?

**Comment by kennethvdberghe** [comment]
I'm not really sure what you mean by that, if I have a look at the files in this exercise, there is no index.html file?

**Comment by kentcdodds** [comment]
I'm speaking more generally. The "index.html" file represents the document HTML the server responds with.

**Comment by kentcdodds** [comment]
I think using the words "html document" may help clear things up.

**Comment by kennethvdberghe** [comment]
Yea, that will prevent users to search for a non-existing file for sure 😃 sorry about the nit-pick, I was just confused.

From: Kent C. Dodds ***@***.***>
Date: Friday, 27 September 2024 at 16:07
To: epicweb-dev/react-suspense ***@***.***>
Cc: Kenneth Van den Berghe ***@***.***>, Author ***@***.***>
Subject: Re: [epicweb-dev/react-suspense] Fixed typos (PR #88)

I think using the words "html document" may help clear things up.

File: exercises/01.fetching/01.problem.throw/README.mdx [hunk 1]
```diff
@@ -9,15 +9,15 @@ gets routed (by the workshop app) through <InlineFile file="api.server.ts" />
 which retrieves the data from `./shared/ship-api-utils.server.ts`.
 
 The problem is that while the `fetch` request is ongoing, the user is just
-staring at a blank white screen. Now we could for sure improve our `index.html`
+staring at a blank white screen. Now we could for sure improve our `index.tsx`
```
Comment (kentcdodds) [inline exercises/01.fetching/01.problem.throw/README.mdx:12]: This should do it.

```suggestion
staring at a blank white screen. Now we could for sure improve the HTML document
```

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**